modules/considerations-scale-down-data-plane.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: REFERENCE
+[id="considerations-scale-down-data-plane_{context}"]
+= Considerations for scaling down the data plane to zero
+
+Review the following considerations before scaling down data plane to zero.
+
+* Use caution when scaling down the data plane to zero. After scaling down the data plane to zero, the workload from the worker nodes disappears.
+
+* The hosted control plane stays up and running. If necessary, you can scale up the `NodePool` resource.
+
+* After scaling down the data plane to zero, some pods in the control plane stay in the `Pending` status.
+
+* After rescaling the `NodePool` resource,  wait for couple of minutes for the `NodePool` resource to become available in a `Ready` state.
+
+* To prevent the pods from draining, you must add the `"machine.cluster.x-k8s.io/exclude-node-draining="` annotation to the worker nodes.
```

Comment:
The Nodes doesn't understand this annotation, it only has an effect on Machines resources. But also let's don't recommend consumers to manipulate internal resources. Instead if you need to skip drain please point to the NodePool drain timeout field

Reply:
Hi @enxebre Can you please re-review the updated PR?